README.md

```diff
@@ -32,7 +32,18 @@ template can not do these for you, otherwise it would :)
 ## RAIL: Redshift Assessment Infrastructure Layers
 
 This package is part of the larger ecosystem of Photometric Redshifts
-in [RAIL](https://github.com/LSSTDESC/RAIL).
+in [RAIL](https://github.com/LSSTDESC/RAIL). This package in particular
+is concerned with wrapping the [LePHARE](https://gitlab.lam.fr/Galaxies/LEPHARE/) template fitting code within RAIL.
+
+### Installation
+
+This package can be installed from source using the following commands:
+```console
+git clone https://github.com/LSSTDESC/rail_lephare.git
+cd rail_lephare
+conda install -c conda-forge pytables sqlite
```

Comment:
I just took a swing at this on my work computer (M1 mac) and I only needed to conda install pytables. I didn't have to include sqlite in the conda install. 

I don't think it hurts anything to include it, but I'm not sure it's necessary.